<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>766</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1180</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="settingScrollArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>348</width>
      <height>456</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100000</width>
      <height>100000</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-278</y>
       <width>325</width>
       <height>732</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="gridPathBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>网格路径</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="pathLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>null path</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="paraSetBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>参数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="BetaLabel">
           <property name="text">
            <string>侧滑角（Beta）：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="BetaInputLine"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="AlphaInputLine"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="ReLabel">
           <property name="text">
            <string>雷诺数（Re）：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="AlphaLabel">
           <property name="text">
            <string>攻角（Alpha）：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <widget class="QRadioButton" name="radioButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>1</y>
              <width>66</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>是</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>1</y>
              <width>66</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>否</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="DimenComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="MaLabel">
           <property name="text">
            <string>马赫数（Ma）：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="MaInputLine">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="ReInputLine"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="compressLabel">
           <property name="text">
            <string>是否可压：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="DimenLabel">
           <property name="text">
            <string>维数：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="countSetBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>计算设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="1">
          <widget class="QLineEdit" name="outStepInputLine"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="recurStepInputLine"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="iterConverInputLine"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="recurStepLabel">
           <property name="text">
            <string>最大迭代步数：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="underRelaxInputLine"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="outStepLabel">
           <property name="text">
            <string>输出步数：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="timeStepInputLine"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="timeStepLabel">
           <property name="text">
            <string>时间步长：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>阻尼因子：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>收敛判据：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="orderComboBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="orderLabel">
           <property name="text">
            <string>计算阶数：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="OutputGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="title">
         <string>折线图输出设置</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="residualCheckBox">
           <property name="text">
            <string>残差</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ClCheckBox">
           <property name="text">
            <string>升力系数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CdCheckBox">
           <property name="text">
            <string>阻力系数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CpCheckBox">
           <property name="text">
            <string>表面压力系数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CfCheckBox">
           <property name="text">
            <string>表面摩擦力系数</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>11</y>
      <width>821</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="reminderLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>303</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>未保存</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSetButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>137</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>保存设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelChangeButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>放弃更改</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建算例</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开算例</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存算例</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>载入网格</string>
   </property>
  </action>
  <action name="actionResidual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>收敛残差</string>
   </property>
  </action>
  <action name="actionCl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>升力系数</string>
   </property>
  </action>
  <action name="actionCd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>阻力系数</string>
   </property>
  </action>
  <action name="actionCp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>表面压力系数</string>
   </property>
  </action>
  <action name="actionCf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>表面摩擦力系数</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
